src/main/java/lotto/domain/AutoLottoGenerator.java

```diff
@@ -0,0 +1,29 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class AutoLottoGenerator implements LottoGenerator {
+    private final Money money;
+
+    public AutoLottoGenerator(int number) {
+        this(new Money(number));
+    }
+
+    public AutoLottoGenerator(Money money) {
+        this.money = money;
+    }
+
+    @Override
+    public List<Lotto> generate() {
+        int cnt = money.getBuyableCount();
+
+        List<Lotto> lottoArray = new ArrayList<>();
+
+        for (int i = 0; i < cnt; i++) {
```

Comment:
```suggestion
        List<Lotto> lottoArray = new ArrayList<>();
        for (int i = 0; i < money.getBuyableCount(); i++) {
```
저는 굳이 불필요한 로컬 변수 사용하지 않는 방식을 선호함
한번 고려해 보면 어떨까?